schemas/com.snowplowanalytics.snowplow/media_player_ad_break/jsonschema/1-0-0

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a context entity, shared with all media_player_ad events belonging to the ad break.",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player_ad_break",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "name": {
+      "description": "Ad break name (e.g., pre-roll, mid-roll, and post-roll).",
+      "type": [
+        "null",
+        "string"
+      ],
+      "maxLength": 4096
+    },
+    "breakId": {
+      "type": "string",
+      "maxLength": 256,
+      "description": "An identifier for the ad break."
+    },
+    "startTime": {
+      "type": "number",
+      "description": "Playback time in seconds at the start of the ad break.",
+      "minimum": 0,
+      "maximum": 9007199254740991
```

Comment:
Why is this max so high?

Reply:
This was because it was based on the previous `media_player` schema which had these limits for playback time. With the new `player` schema, I have changed this now to 2147483647.